// let b = 2;

try {
  let a = 2 + b;
} catch (e) {
  console.log(`Algo deu errado -> ${e}`);
}

// Algo deu errado -> ReferenceError: b is not defined
